### PR TITLE
Backend template

### PR DESCRIPTION
**Pull request by SKairinos**
This change is [Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-template-backend/9)

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#
+# 1. Uncomment the git-codeforlife package under [packages].
+# 2. Uncomment the git-codeforlife package under [dev-packages].
+# 3. Comment out the editable-codeforlife package under [dev-packages].
+# 4. Set the ref key of the git-codeforlife packages under [packages] and
+#     [dev-packages] to have the same version number.
+# 5. Run `pipenv install --dev` in your terminal.
+
+[packages]
+codeforlife = {ref = "backend_template", git = "https://github.com/ocadotechnology/codeforlife-package-python.git"}
```
Comment (SKairinos) [inline Pipfile]: install new version

**Review by faucomte97 — CHANGES_REQUESTED** [review]
> Reviewed 91 of 91 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-template-backend/9)* status: all files reviewed, 2 unresolved discussions (waiting on @SKairinos)

___
*[`api/models/fruit.py` line 24 at r1](https://reviewable.io/reviews/ocadotechnology/codeforlife-template-backend/9#-O0xWWvjDsIo65_DtjrN:-O0xWWvjDsIo65_DtjrO:bgg4ako) ([raw file](https://github.com/ocadotechnology/codeforlife-template-backend/blob/68780adcb665dfc15c72c2c1de405e66099e906c/api/models/fruit.py#L24)):*
> ```Python
>     is_citrus = models.BooleanField(_("is citrus"))
> 
>     expires_on = models.DateField(_("expires on"))
> ```

no new lines in between fields pls



_Code quote:_
```Python
    name = models.CharField(_("name"), max_length=50)

    is_citrus = models.BooleanField(_("is citrus"))

    expires_on = models.DateField(_("expires on"))
```

**Review by faucomte97 — COMMENTED** [review]
> Reviewed 5 of 5 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-template-backend/9)* status: all files reviewed, 2 unresolved discussions (waiting on @SKairinos)

**Review by SKairinos — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-template-backend/9)* status: 90 of 91 files reviewed, 2 unresolved discussions (waiting on @faucomte97)

___
*[`api/models/fruit.py` line 24 at r1](https://reviewable.io/reviews/ocadotechnology/codeforlife-template-backend/9#-O0xWWvjDsIo65_DtjrN:-O0y5DyR66UC0WpFY0DQ:b-896fix) ([raw file](https://github.com/ocadotechnology/codeforlife-template-backend/blob/68780adcb665dfc15c72c2c1de405e66099e906c/api/models/fruit.py#L24)):*
<details><summary><i>Previously, faucomte97 (Florian Aucomte) wrote…</i></summary><blockquote>

no new lines in between fields pls
</blockquote></details>

Done.

**Review by faucomte97 — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-template-backend/9#-:-O0y5Iin7ALwQxqi27Wp:bnfp4nl)* status: all files reviewed, 1 unresolved discussion

**Pull request merged**